backend/glfwbackend/glfw_backend.cpp

```diff
@@ -115,7 +115,7 @@ GLFWwindow *igCreateGLFWWindow(const char *title, int width, int height,
 
   // Install extra callback
   glfwSetWindowRefreshCallback(window, glfw_window_refresh_callback);
-  glfwMakeContextCurrent(NULL);
```

Comment:
I think this is expected for multi window scenerio (?)
I think that you should keep this and use analogical method in your project and pass `(*GLFWBackend).handle()` (could make public if necessary)

Reply:
This one allow to call OpenGL function after creating window, like compile shader. This behaviour was valid in SDL backend. Otherwise you can only call OpenGL function on render loop or inside `AfterCreateContext` callback, since OpenGL function need `glfwMakeContextCurrent` to be set.

Reply:
hmm, ok so remove that line instead of commenting it out.

Reply:
done

Reply:
I've investigated a bit more to justify this:
it causes because giu uses mainthread to call Run() in mainthread.
possibly caused by the fact that CreateWindow might be called in another goroutine than Render loop

Reply:
@gucio321 sure, sorry for the inconvenience.